OlexanderHramatovych/HW7/Task3.py

```diff
@@ -0,0 +1,7 @@
+# creating function which counts letters in word
+def number(word):
```

Comment:
good job 
look at this version of code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result